intelligence/signals.py

```
@@ -0,0 +1,40 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from laboratory.models import LabResult
+from encounters.models import Encounter
+from .tasks import run_anomaly_detection_for_new_lab, run_pattern_analysis_for_patient
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=LabResult)
+def trigger_anomaly_detection_on_new_lab(sender, instance, created, **kwargs):
```

Ruff (0.12.2)

Missing type annotation for `**kwargs`

(ANN003)